ads/mediaimpact.js

```diff
@@ -25,6 +25,15 @@ import {loadScript} from '../3p/3p';
 
 export function mediaimpact(global, data) {
   global.fif = false;
+  /* eslint google-camelcase/google-camelcase: 0 */
+  global.sas_loadHandler = function(f) {
+    if (!f.hasAd) {
+      global.context.noContentAvailable();
+    } else {
+      //global.context.requestResize(f.crea1.width, f.crea1.height);
```

Comment:
doesn't minify remove this automatically for prod? I put this line there because it isn't working. I tried locally without success using ads example page so I put it there for review.